x-pack/plugins/fleet/server/services/elastic_agent_manifest.ts

```diff
@@ -73,6 +73,13 @@ spec:
               value: "/etc/elastic-agent"
           securityContext:
             runAsUser: 0
+            capabilities:
+              add:
+              # The following capabilities are needed for 'Defend for containers' integration (cloud-defend)
+              # If you are not using this integration, then these capabilites can be removed.
```

Comment:
@elastic/cloud-security team opened this based on this https://github.com/elastic/elastic-agent/pull/2283